tokens.xml

```diff
@@ -15306,7 +15306,7 @@ Whenever an opponent draws a card, this emblem deals 1 damage to them.</text>
                 <maintype>Emblem</maintype>
             </prop>
             <set picURL="https://cards.scryfall.io/large/front/2/c/2c4c9934-fe82-4560-b0bf-0ebc863ff0d1.jpg?1721426368">BLB</set>
-            <reverse-related/>
+            <reverse-related></reverse-related>
```

Comment:
Ha! This highlighted a left out relation in #276 

Since we do not validate or check the file for empty relations etc. it stayed unnoticed.

Will open a fix and link to [Ral, Crackling Wit](https://scryfall.com/card/blb/230/ral-crackling-wit).

Reply:
cool, I should maybe check for that before merging